simulator/field.py

```diff
@@ -2,70 +2,29 @@
 """
 File:          Field.py
 Author:        Moungsung Im 
-Last Modified: Binit on 10/30
+Last Modified: Ammar on 11/06
 """
 
 import pybullet as p
 
-from simulator.buttons import Buttons
 from simulator.utilities import Utilities
+from simulator.field_components.buttons import Buttons
```

Comment:
Get rid of the `field_components` module for this PR. We'll do another one where we split into two modules: `environment` and `agent`.